android/build.gradle

```
@@ -75,7 +75,7 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.annotation:annotation:1.6.0'
+    implementation 'androidx.annotation:annotation:1.3.0'
```

Helpful comment for future us maybe?

```suggestion
    // TODO: Upgrading to 1.6.0 depends on kotlin stdlib 1.8 which breaks the build. Upgrade once we're on Kotlin 1.8.
    implementation 'androidx.annotation:annotation:1.3.0'
```